android/src/newArch/java/com/learnium/RNDeviceInfo/RNDeviceModule.java

```diff
@@ -0,0 +1,650 @@
+package com.learnium.RNDeviceInfo;
```

Comment:
Do we really want to do it this way? Can't we just add the `NativeRNDeviceInfoAndroidSpec` on `paper` too and have the single implementation without the need to duplicate all of this?

It's done like this e.g. in `react-native-svg` where we have the `Spec` in `paper` dir: https://github.com/software-mansion/react-native-svg/blob/a36a676d4313225bbcec44120eb38ce871106d0a/android/src/paper/java/com/horcrux/svg/NativeSvgRenderableModuleSpec.java and single implementation: https://github.com/software-mansion/react-native-svg/blob/a36a676d4313225bbcec44120eb38ce871106d0a/android/src/main/java/com/horcrux/svg/RNSVGRenderableManager.java#L32

Reply:
We have a single implementation in `RNDeviceModuleManager` we're just calling through from here. I'm not sure it's much better to duplicate the interface. You still need to manually keep it in sync with the spec. I've made the change anyway